marlowe-playground-client/src/Analytics.purs

```diff
@@ -7,7 +7,6 @@ import Data.Undefinable (Undefinable, toUndefinable)
 import Data.Unit (Unit)
 
 foreign import trackEvent_ ::
-  forall eff.
```

Comment:
I think we need to find a way to get rid of the warnings from the generated code so that we can see warnings like this (and possibly fail on warn)